spec/lib/dispatch-rider/logging/json_formatter_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe DispatchRider::Logging::JsonFormatter do
+  let(:fs_message) { DispatchRider::Message.new(subject: 'test', body: { key: 'value' }) }
+  let(:item) { nil }
+  let(:queue) { nil }
+  let(:message) { DispatchRider::QueueServices::FileSystem::FsReceivedMessage.new(fs_message, item, queue) }
```

Comment:
is that meant to wrap around the message? I thought its meant to be a File object ..